web/profiles/custom/yalesites_profile/composer.json

```diff
@@ -45,7 +45,7 @@
         "drupal/redirect": "^1.7",
         "drupal/twig_tweak": "^3.1",
         "drupal/xmlsitemap": "^1.2",
-        "yalesites-org/atomic": "dev-YALB-476-wire-image-paragraph-artifact"
+        "yalesites-org/atomic": "dev-main"
```

Comment:
We're currently creating "artifact" branches manually, and pushing to Pantheon (since pantheon can run composer, but not npm install). We're working on getting the dev-ops stuff in place to handle this for us, but it's not ready yet. As far as I can tell, the multidev is showing everything correctly. Did you notice errors on the multidev?